motr/setup.c

```diff
@@ -1155,6 +1155,8 @@ static int reqh_context_services_init(struct m0_reqh_context *rctx,
 		if (rctx->rc_services[i] == NULL ||
 		    M0_IN(i, (M0_CST_HA, M0_CST_SSS)))
 			continue;
+#if 0
```

Comment:
If this is commented out, how is the FIS started on server side?

So we have to add FIS services into the conf.xc?